### DISCUSSION TITLE
Saving a Rotated Image back to Server

### DISCUSSION THREAD

**Discussion by crmconsultancy** [sachinchoolur/lightGallery#1336] [Q&A]
Hi there,

I'm evaluating LightGallery for use in a customer application to allow that customer rotate images and then save back into their server storage.

So far so good as the rotate and flip functionality works grand.

However I was wondering if anyone had an example of how to save the edited image back to a server endpoint.

From my research so far, this appears to be one of two options:
(1) There is a webhook in LightGallery that I've not found yet and this may allow a direct communication from a LightGallery Property to a .ashx or rest endpoint to save the image.
(2) Alternatively we should get the image DOM Element as a Canvas object and then save from there.

But wanted to reach out and see if I am missing an easier way using LightGallery before getting stuck into the raw DOM manipulation.

Kind Regards, Paul.

**Comment by sachinchoolur** [comment]
Hey @crmconsultancy ,

Yes, it is possible. You can use the `lgRotateLeft`, and `lgRotateRight` events to get notified when users rotate the images. 

Docs- https://www.lightgalleryjs.com/docs/events/#lgRotateLeft

If you face any issues, please let me know
